release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1598,6 +1598,156 @@ If these values are not defined in a failure domain configuration, for instance
+
+* Previously, during CAPI installer-provisioned installation, the user provided a `machineNetwork` parameter. With this release, the installer uses a random `machineNetwork` parameter.
+(link:https://issues.redhat.com/browse/OCPBUGS-45485[*OCPBUGS-45485*])
+
+* Previously, during an installation on {aws-first}, the installer used the wrong load balancer when searching for the `hostedZone` ID, which caused an error. With the release, the correct load balancer is used and the issue is resolved.
+(link:https://issues.redhat.com/browse/OCPBUGS-45301[*OCPBUGS-45301*])
+
+* Previously, endpoint overrides in {ibm-power-server-title} were not conditional. As a consequence, endpoint overrides were created incorrectly and caused failures in Virtual Private Environments (VPEs). With this release, endpoint overrides are conditional only for disconnected installations.
+(link:https://issues.redhat.com/browse/OCPBUGS-44922[*OCPBUGS-44922*])
+
+* Previously, during a shared Virtual Private Cloud (VPC) installation, the installer added the records to a private DNS zone created by the installer instead of adding the records to the cluster’s private DNS zone. As a consequence, the installation failed. With this release, the installer searches for an existing private DNS zone and, if found, pairs that zone with the network that is supplied by the `install-config.yaml` file and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-44641[*OCPBUGS-44641*])
+
+* Previously, the `oc adm drain --delete-local-data` command was not supported in the 4.18 `oc` cli tool. With this release, the command has been updated to `oc adm drain --delete-emptydir-data`. (link:https://issues.redhat.com/browse/OCPBUGS-44318[*OCPBUGS-44318*])
+
+* Previously, US East (`wdc04`), US South (`dal13`), Sydney (`syd05`), and Toronto (`tor01`) regions were not supported for {ibm-power-server-title}. With this release, these regions, which include `PowerEdgeRouter` (PER) capabilities, are supported for {ibm-power-server-title}.
+(link:https://issues.redhat.com/browse/OCPBUGS-44312[*OCPBUGS-44312*])
+
+* Previously, during a {gcp-first} installation, when the installer was creating filters with large numbers of returned data, for example for subnets, it exceeded the quota for the maximum number times that a resource can be filtered in a specific period. With this release, all relevant filtering is moved to the client so that the filter quotas are not exceeded and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-44193[*OCPBUGS-44193*])
```

Comment:
```suggestion
* Previously, during a {gcp-first} installation, when the installation program was creating filters with large numbers of returned data, for example for subnets, it exceeded the quota for the maximum number times that a resource can be filtered in a specific period. With this release, all relevant filtering is moved to the client so that the filter quotas are not exceeded and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-44193[*OCPBUGS-44193*])
```